src/api/services/summary.py

```diff
@@ -1,27 +1,64 @@
-from _main_.utils.common import serialize_all
+from _main_.utils.common import serialize, serialize_all
 from _main_.utils.footage.spy import Spy
 from _main_.utils.massenergize_errors import MassEnergizeAPIError
 from _main_.utils.massenergize_response import MassenergizeResponse
 from api.store.summary import SummaryStore
 from typing import Tuple
 
+
 class SummaryService:
-  """
-  Service Layer for all the summaries
-  """
+    """
+    Service Layer for all the summaries
+    """
+
+    def __init__(self):
+        self.store = SummaryStore()
+
+    def next_steps_for_admins(
+        self, context, args
+    ) -> Tuple[tuple, MassEnergizeAPIError]:
+        content, err = self.store.next_steps_for_admins(context, args)
+
+        if err:
+            return None, err
+
+        last_visit = content.get("last_visit")
+        testimonials = content.get("testimonials", [])
+        messages = content.get("messages", [])
+        team_messages = content.get("team_messages", [])
+        users = content.get("users", [])
+        teams = content.get("teams", [])
+
+        # testimonials = serialize_all(testimonials)
+        # messages = serialize_all(messages)
+        # users = serialize_all(users)
+        # teams = serialize_all(teams)
 
-  def __init__(self):
-    self.store =  SummaryStore()
+        content = {
```

Comment:
Seems like you fill content in the Store layer method, and then don't check it but fill it again here.